doc/asab/web/restapidocs.rst

```diff
@@ -0,0 +1,107 @@
+REST API Docs
+==============
+
+ASAB's API service generates a `Swagger documentation <https://swagger.io/specification>`_ which automatically shows all
+of your endpoints. However you will need to follow some steps to add summaries, tags and more.
+
+Before you start, make sure that you have a `REST API microservice <https://asab.readthedocs.io/en/latest/tutorial/web/chapter2.html>`_.
+
+If you want Authorization in Swagger, you will need a OpenIDConnect endpoint.
+
+In your microservice
+-----------
+
+First you need to initialize the API Service:
+
+.. code-block:: python
+
+    # Initialize API service
+    self.ApiService = asab.api.ApiService(self)
+
+    # Introduce Web to API Service
+    self.ApiService.initialize_web()
+
+After initializing the API Service a **/doc** endpoint will become available. You will notice
+that some or none of your endpoints have summaries, tags or descriptions.
+
+That's because you need to add a docstring to your endpoint method:
+
+.. code-block:: python
+
+    async def endpoint(self, request):
+        """
+        Summary looks like this and takes the first line from docstring.
+
+        Description of what this endpoint does.
+
+        ---
+        tags: [asab.mymicroservice]
+        """
+
+The first line of the docstring gets shown as a summary next to the endpoint in Swagger.
+There are also extra parameters you can set like tags, which work like categories.
+
+The description of the microservice also needs a docstring for it to show up:
+
+.. code-block:: python
+
+    class TutorialApp(asab.Application):
+        """
+        TutorialApp is the best microservice in the world!
+
+        <marquee>The description supports HTML tags</marquee>
+        """
+
+Authorization in Swagger
+-----------
+Authorization requires having an OpenIDConnect endpoint with an Authorization and Token endpoint (like SeaCat).
```

Comment:
having it where? in my microservice, right?
It is ASAB documentation and there's no need to know or use SeaCat. It is called SeaCat Auth, you should provide link to github and write like one sentence about why you mention SeaCat Auth here.

Reply:
Well, the endpoint doesn't need to be inside the microservice, because the OpenIDConnect endpoint is not made by the microservice but SeaCat Auth or other OpenIDConnect software